packages/mobile_app/lib/main.dart

```diff
@@ -63,10 +63,11 @@ void main() async {
 
   await Future.wait([
     notificationFuture,
-    assets
+    assets,
+    initAd,
```

Comment:
I disagree we need to wait for an ad to be loaded before start drawing

Reply:
Nothing is drawn before BuildContext (elements tree) is created.